packages/contracts-frontend/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@uma/contracts-frontend",
+  "version": "0.1.0",
+  "description": "UMA smart contracts and unit tests",
+  "devDependencies": {
+    "@uma/core": "^2.5.0",
```

Comment:
so the magic here, really, is that this is a `devDependency`, right? If this code is generated dynamically like this do you really need to include these dependencies anyways? 

I guess you do seeing you've got the logic for generating ts and the build within this package.json rather than within core or common so that does make sense. if you placed the generation logic in one of _those_ packages then we would not need either of these dependencies, right?

However, it might make more sense the way you have it now as this'll let you use core logic within unit tests/integration test from front-end dependents but can shake away all unused stuff in production.

Reply:
Yeah, we need to explicitly provide the deps here to tell lerna that those packages must be built before this one. Otherwise the artifacts might not exist when we start building. ethers is also required because the typescript compilation fails if you don't have it I think.

Reply:
But to your initial question, yes. The dev dependencies mean that none of these should get packaged up in the final bundle.